name.abuchen.portfolio/src/name/abuchen/portfolio/money/currencies_fr.properties

```diff
@@ -337,6 +337,16 @@ VUV = Vatu
 
 WST = Tala
 
+XAF = Franc CFA (CEMAC)
+
+XCD = Dollar des Cara\u00EFbes orientales
+
+XDR = FMI Droits de tirage sp\u00E9ciaux
```

Comment:
French is a language where determiners are put behind the main word and often require a preposition. (Can be seen in “inverted abbreviations”: e.g. NATO, North Atlantic Treaty Organization, is OTAN in French, l’organisation du traité de l’Atlantique du Nord; or UNO, United Nations Organization, is ONU, l’organisation des nations unies). So I suggest “Droits de tirage spéciaux du FMI” here.

Reply:
Thank you. I fixed it.